src/util/prolog/prologSession.ts

```diff
@@ -29,6 +40,14 @@ function resolveImports(program: string, defaultPath: string, currentFileName: s
 		return importedPrograms + program;
 }
 
+/**
+	* Creates the dependency graph for imports of the prolog files recursively
+	* @param {string} program - The program code to look for imports
+	* @param {string} defaultPath - The path in which to look for the file names
```

Comment:
Does this result in a static path? Designwise this could later be adapted to be a dynamic path.

Reply:
Edit: Didn't see the other variable relativePath earlier

Reply:
Would you think this is clear then or should the naming be changed still?

Reply:
staticPath might also be viable, but defaultPath is just as understandable, So you can leave it be.